src/components/BookPages.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import SingleBook from './SingleBook';
+import BookForm from './BookForm';
+
+const BookPages = () => (
+  <div>
+    <SingleBook />
```

Comment:
- [ ] It may be a good idea to have a component or a piece of code that shows a list of books and It should use your SingleBook component to render a book. 😉